application/src/main/java/com/sanction/lightning/facebook/FacebookProvider.java

```diff
@@ -27,4 +53,61 @@ public FacebookUser getFacebookUser() {
     return client.fetchObject("me", FacebookUser.class, Parameter.with("fields",
             "first_name, last_name, middle_name, gender, name, verified"));
   }
+
+  /**
+   * Gets the users photos.
+   * @return an array of URLs representing the users photos
+   */
+  public List<String> getFacebookUserPhotos() {
+    JsonObject photos = client.fetchObject("me/photos", JsonObject.class,
+            Parameter.with("type", "uploaded"), Parameter.with("fields", "link"));
+    JsonArray photosArray = photos.getJsonArray("data");
```

Comment:
This line is a little confusing if someone is trying to just read through it - can you add a comment for this line to explain what it does?